data/translations/en/months.json

```diff
@@ -1,14 +1,14 @@
 {
-  "1": "january",
-  "2": "february",
-  "3": "march",
-  "4": "april",
-  "5": "may",
-  "6": "june",
-  "7": "july",
-  "8": "august",
-  "9": "september",
-  "10": "october",
-  "11": "november",
-  "12": "december"
+  "january": "january",
+  "february": "february",
+  "march": "march",
+  "april": "april",
+  "may": "may",
+  "june": "june",
+  "july": "july",
+  "august": "august",
+  "september": "september",
+  "october": "october",
+  "november": "november",
+  "december": "december"
```

Comment:
Since we are changing from a numeric month to name we need to:
- Update our translation template to reflect this
- Update our template processing documentation to make sure that this change is made for incoming translations.

Reply:
The translation template already uses the month names but I updated the processing documentation to not delete that column now